build_scripts/build_win_license_dir.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# PULL IN LICENSES USING NPM - LICENSE CHECKER
+npm install -g license-checker
+
+cd ../chia-blockchain-gui || 'exit" or "cd' ... || return
```

Comment:
Same thing here about interesting quotations but with the addition of an ellipses

Reply:
These are a carryover. I changed it to "exit 1".